source/TrainEditor2/Extensions/Utilities.cs

```diff
@@ -8,6 +8,14 @@
 
 namespace TrainEditor2.Extensions
 {
+	internal enum NumberRange
```

Comment:
I'd be tempted to move the number parsing utilities into the API.

In the existing parsers, parsing and validating takes up quite a bit of code, and could definitely be simplified into global helper functions.
Not saying do that now, but as you've written these functions, it might be an idea to think about it so that they could be dropped in as changes are made?

Reply:
Thank you.
The latest code has been modified to the suggested code. (Not committed yet.)

Reply:
I had a misunderstanding.
If you move to the API, I will follow it.